Packages/com.unity.inputsystem/InputSystem/State/InputStateBlock.cs

```diff
@@ -42,27 +42,27 @@ public unsafe struct InputStateBlock
         public const uint AutomaticOffset = 0xfffffffe;
 
         // Primitive state type codes.
-        public static FourCC kTypeBit = new FourCC('B', 'I', 'T');
-        public static FourCC kTypeSBit = new FourCC('S', 'B', 'I', 'T');
-        public static FourCC kTypeInt = new FourCC('I', 'N', 'T');
-        public static FourCC kTypeUInt = new FourCC('U', 'I', 'N', 'T');
-        public static FourCC kTypeShort = new FourCC('S', 'H', 'R', 'T');
-        public static FourCC kTypeUShort = new FourCC('U', 'S', 'H', 'T');
-        public static FourCC kTypeByte = new FourCC('B', 'Y', 'T', 'E');
-        public static FourCC kTypeSByte = new FourCC('S', 'B', 'Y', 'T');
-        public static FourCC kTypeLong = new FourCC('L', 'N', 'G');
-        public static FourCC kTypeULong = new FourCC('U', 'L', 'N', 'G');
-        public static FourCC kTypeFloat = new FourCC('F', 'L', 'T');
-        public static FourCC kTypeDouble = new FourCC('D', 'B', 'L');
+        public static readonly FourCC kTypeBit = new FourCC('B', 'I', 'T');
```

Comment:
Need to rename all these to remove the `k`. Will do that in a follow up PR where I'd add the const naming test coverage to include `static readonly` fields.